app/components/register/register.controller.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+    'use strict';
+
+    angular.module('FoodOrderingApp')
+        .controller('RegisterController', RegisterController);
+
+    RegisterController.$inject = ['$state', '$log', '$timeout', 'UserService', 'APP_CONSTANT'];
+
+    function RegisterController($state, $log, $timeout, UserService, APP_CONSTANT) {
+
+        var vm = this;
+
+        vm.firstNameMsg = APP_CONSTANT.FIRST_NAME_MSG;
+        vm.lastNameMsg = APP_CONSTANT.LAST_NAME_MSG;
+        vm.emailMsg = APP_CONSTANT.EMAIL_MSG;
+        vm.invalidEmailMsg = APP_CONSTANT.INVALID_EMAIL_MSG;
+        vm.contactNumMsg = APP_CONSTANT.CONTACT_NO_MSG;
+        vm.addressMsg = APP_CONSTANT.ADDRESS_MSG;
+        vm.passwordMsg = APP_CONSTANT.PASSWORD_MSG;
+        vm.confirmPasswordMsg = APP_CONSTANT.CONFIRM_PASSWORD_MSG;
+
+        vm.user = {};
+        vm.dataLoading = false;
+        vm.errorStatus = false;
+
+        vm.style = '1px solid red;';
```

Comment:
do not use css in the controller, instead create separate class for it and use ng-class